Modules/Office365DSC/Modules/Office365DSCUtil.psm1

```diff
@@ -1462,16 +1462,16 @@ function Test-PnPOnlineConnection
     (
         [Parameter(Mandatory = $true)]
         [System.String]
-        $SiteUrl,
+        $SPOCentralAdminUrl,
```

Comment:
This has to be SiteURL, not SPOCentralAdminUrl. PnP let's you establish a connection with any site collection, not just Central Admin

Reply:
Done.